bundles/framework/publisher2/tools/LayerSelectionTool.js

```diff
@@ -50,6 +58,7 @@ Oskari.clazz.define('Oskari.mapframework.publisher.tool.LayerSelectionTool',
                     pluginConfig.config.baseLayers = layerSelection.baseLayers;
                     pluginConfig.config.defaultBaseLayer = layerSelection.defaultBaseLayer;
                 }
+                pluginConfig.config.isStyleSelectable = me.__plugin.getStyleSelectable();
```

Comment:
Shouldn't this come in the this.getPlugin().getConfig() call above?

Reply:
`getConfig()` does not return the changes made in the publisher to the plugin config. I guess it returns the initial config given to the plugin. That's why `baseLayers` and `defaultBaseLayer` is written separately, too, on the above lines.